modules/salt/pages/formula-saltboot.adoc

```diff
@@ -107,66 +107,21 @@ Partition Flags: boot
 
 Leave the other fields empty.
 
+[[saltboot-formula-disk-selection]]
+== Disk selection in saltboot formula
 
+When there is only one disk present on target hardware (including USB drives), an asterisk [systemitem]``*`` can be used to automatically select the disk device.
 
-== Customize the Client Boot Process
+If selecting exact devices is preferred, disk device field can be in format of symbolic names (e.g. [path]``/dev/sda``), by-path (e.g. [path]``/dev/disk/by-path/..``) or by-id (e.g. [path]``/dev/disk/by-id/...``)
 
```

Comment:
/dev/disk/by-path/ prefix is used by default, if the name does not contain any /
so /dev/disk/by-path/*usb* is the same as *usb*

List of available devices is visible on the terminal if you press ESC during waiting for key approval.

Reply:
Updated in latest commit.